### PR TITLE
Added documentation for the new Swift API

### PR DESCRIPTION
**Pull request by kapitoshka438**

### CODE REVIEW

File: docs-src/v2/content/audio-input/_index.md [hunk 1]
```diff
@@ -166,6 +193,36 @@ print("confidence:", result.confidence)
...
+// When there is no more audio data to send
+// (i.e. when the app needs to stop recording audio)
+// call ASRStream's result() method to close the ASR stream
+self.asrStream?.result(completion: { (error) in
+	if let error = error {
+		print(error.localizedDescription)
+	}
+})
+
+// ASR Results come to asrResultHandler block on creating ASRStream
```
Comment (mplooster) [inline docs-src/v2/content/audio-input/_index.md:206]: I'm not sure I understand from the example how the above call to `self.asrStream?.result(...)` relates to this call to create the ASR stream. Swift users must call the result method before creating the stream? Is this a construct that most swift users will understand?

**Review by mplooster — APPROVED** [review]
Looks pretty good! I have one comment below, which might be due to my lack of understanding of Swift.

File: docs-src/v2/content/audio-input/_index.md (hunk 1 above)
Comment (kapitoshka438) [reply docs-src/v2/content/audio-input/_index.md:206]: No, it's my fault. ASRStream has to be created first. When a developer creates a stream (i.e. when he starts recording, it handles ASR results right in the completion block of this function. 
Then, when recording is stopped, he should call `result` method to close the ASR stream. I will swap these two methods.

Comment (kapitoshka438) [reply docs-src/v2/content/audio-input/_index.md:197]: @mplooster fixed.

Comment (mplooster) [reply docs-src/v2/content/audio-input/_index.md:197]: Thanks!

**Review by mplooster — APPROVED** [review]

**Pull request merged**